CHANGELOG.md

```diff
@@ -3,6 +3,23 @@ Changelog for NeoFS LOCODE database
 
 ## [Unreleased]
 
+## [0.7.0] - 2024-08-28
+
+### Changed
+- Go 1.22 is required to build now (#40, #43)
+- Added a file with which you can overwrite the coordinates of incorrect 
+records (#38)
+- Drop cobra dependency, so there is no CLI (#42)
+
+### Updated
+- UN/LOCODE to `2024-1` version (#44)
+- UN/LOCODE files from gitHub (#37)
```

Comment:
More like 'switch data source to github.com/datasets/un-locode which has more suitable data format'.